src/components/PopoverWithMeasuredContent.js

```diff
@@ -27,10 +27,6 @@ const propTypes = {
         vertical: PropTypes.oneOf(_.values(CONST.MODAL.ANCHOR_ORIGIN_VERTICAL)),
     }),
 
-    /** A function with content to measure. This component will use this.props.children by default,
-    but in the case the children are not displayed, the measurement will not work. */
-    measureContent: PropTypes.func.isRequired,
```

Comment:
It's clear to me that this was not needed. It also said it would use this.props.children by default, but it did not. Everywhere this component was used we had `measureContent` be an exact or near-exact duplicate of the component's children.

Furthermore, children and measureContent were swapped, such that the children would be rendered invisibly + measured, then whatever was returned by measureContent would be rendered visibly with the adjusted position.